Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml

```diff
@@ -341,10 +341,17 @@
 
 # Pills
 - type: entity
-  name: dexalin pill (10u)
+  name: pill (dexalin 10u)
   parent: Pill
   id: PillDexalin
   components:
+  - type: Pill
+    pillType: 15
+  - type: Sprite
+    sprite: Objects/Specific/Chemistry/pills.rsi
```

Comment:
i think you can remove that line since its written in parent

Reply:
Fair point. I removed it from all pills, including the charcoal pill that already had the line.